src/app/code/community/AvS/FastSimpleImport/Model/Import/Entity/Product.php

```diff
@@ -1331,6 +1343,9 @@ public function validateRow(array $rowData, $rowNum)
         $this->_validatedRows[$rowNum] = true;
 
         if (isset($this->_newSku[$rowData[self::COL_SKU]])) {
+	    if($this->getIgnoreDuplicates()){
+		return true;
+	    }
             $this->addRowError(self::ERROR_DUPLICATE_SKU, $rowNum);
```

Comment:
Should we return here? I my opinion we should just not throw the error and validate the row further?